src/test/converter/decorators/decorators.ts

```diff
@@ -43,6 +43,6 @@ function decoratorWithParam(value:boolean):MethodDecorator {
  */
 function decoratorWithOptions(options:{name:string}):ClassDecorator {
     return function (target) {
-        target.options = options;
+        target.arguments = target;
```

Comment:
So there error here is caused by a lib.es5.d.ts definition. Granted, this is only an example being documented but I'd prefer to update the definition of the target function to include this property.